webapp/app/help.html

```diff
@@ -546,6 +546,35 @@
         </div>
     </div>
 </div>
+<div class="col-xs-12">
+    <div class="panel panel-default">
+            <div class="panel-heading"><h3>URL Query String Parameters</h3></div>
+            <div class="panel-body panel-spacing">
+                <table id="shortcuts">
+                    <tr>
+                      <td><span class="kbd">nojobs</span></td>
+                      <td>Load result sets and revisions without any job results
+                    </tr>
+                    <tr>
+                      <td><span class="kbd">fromchange</span></td>
+                      <td>Specify the earliest revision hash in the resultset range
+                    </tr>
+                    <tr>
+                      <td><span class="kbd">tochange</span></td>
+                      <td>Specify the latest revision hash in the resultset range
+                    </tr>
+                    <tr>
+                      <td><span class="kbd">startdate</span></td>
+                      <td>Specify the earliest date in the resultset range (2015-02-18)
```

Comment:
Maybe we shouldn't hard code a date here, and below in the description.

But I agree the fromchange, tochange, startdate, endate could provide an example in the url param field on the left side of the table. For that date ones, maybe a "January 1st, 2015" to "January 1st, 2020" type of date range, so we don't have to keep updating the example.